client/src/components/UIManager.js

```diff
@@ -34,11 +35,13 @@ export class UIManager {
 		this.content.fixtures = document.getElementById('fixtures');
 		this.content.standings = document.getElementById('standings');
 		this.content.standingsTable = document.getElementById('standingsTable');
+		this.content.gameInformation = document.getElementById('gameInformation');
```

Comment:
I like this idea to remove this content from the 2d canvas, we could even just adjust the css/html element to make it overlay the game